openfisca_core/periods.py

```diff
@@ -841,27 +847,16 @@ def raise_error(value):
     return Period((unit, base_period.start, size))
 
 
-def compare_period_size(a, b):
-    unit_a, start_a, size_a = a
-    unit_b, start_b, size_b = b
+def key_period_size(period):
```

Comment:
Add a comment to explain the role of this function and its return format? In this context, the reader might not understand what `key` refers to.

Reply:
@fpagnoux In what case in size > 1? in set_inputs ?

Reply:
Yes, for instance you could set a `salary` for 3 years (2012 to 2014), then the unit would be YEAR and the size 3.  
This is indeed only possible if a `set_input` has been defined for the variable.